rules/cross-platform/execution_python_script_in_cmdline.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2021/01/13"
+maturity = "development"
+updated_date = "2021/01/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a Python script is executed using command line input. Attackers often use this method to execute
+malicious scripts and avoiding writing it to disk.
```

```suggestion
Identifies when a Python script is executed using command line input and imports the sys module. Attackers often use this method to execute
malicious scripts and avoiding writing it to disk.
```

This would also need a relint